.github/workflows/push-to-prod-draft.yml

```diff
@@ -9,39 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sbc: [raspi, rockpi, rak, pisces, og, sensecap, finestra, controllino, cotx]
```

Comment:
Why are all of the third party mainnet and testnet being removed?

Reply:
That was the one of the acceptance criteria of the issue. Should I enable all again?

Reply:
I'm not sure why we would remove the other brands? Are we moving them to an action somewhere else?

Reply:
@shawaj I think that part of the AC was a bit missleading as the ticket didn't imply changing the actual deployment workflow. It was supposed to still deploy updates to the existing OpenFleets

Reply:
OK no worries, I'm bringing all back in the new style.